cmd/virt-launcher/entrypoint.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+set +e
```

Comment:
I don't think we can use 'set -e' here.

We're doing our own error checking later on in this file

Reply:
Oh it is `set +e`. I wanted to make explicit that we do custom error checking in that script.

Reply:
i should finish my morning coffee before doing reviews

Reply:
Isn't `set +e` explicitly not erroring on errors?